Fix build with perl 5.16+ due to missing ctime.pl:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=724193
https://github.com/dankelley/gri/commit/58949a199d4b307c8c95751a42ccabe22073d824
And getopts.pl:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=735914
https://github.com/dankelley/gri/commit/8f6e468636d7d9b32945845f408044af0e095fb7
--- doc/texinfo2HTML.orig
+++ doc/texinfo2HTML
@@ -54,18 +54,17 @@ BUGS:
 		the end is here} and more blah
 	will not get the \@code{} item converted correctly.
 ";
-require "ctime.pl";
-require "getopts.pl";
-
-$debug = 0;
-
-die if !&Getopts('h');
-$print_help = 0;
-$print_help = $opt_h if $opt_h;
-if ($print_help) {
+#require "ctime.pl";
+use POSIX qw(ctime);
+use Getopt::Std;
+&getopts('h');
+if ($opt_h) {
     print "$usage";
     exit 0;
 }
+$debug = 0;
+
+#$date = &ctime(time);
 $date = &ctime(time);
 chop($date);
 
